### ISSUE TITLE
Create a cumulative to incident converter

### ISSUE THREAD

**Issue by sabinala** [ciemss/pyciemss#574]
Create a function to convert ensemble model samples of cumulative state variables (i.e., cases, hospitalizations, and deaths) to incident values so that they can be included in CDC formatting and results.

**Comment by sabinala** [comment]
Closing as this will not be necessary once we can use sampled parameter values to define observables. (Subsumed by #578)

**Comment by SamWitty** [comment]
I'm not sure I see the connection between this and #578 . My understanding is that this issue is necessary because our observables can't aggregate over time. Addressing #578 does not necessarily address aggregating observables in time; instead it only allows for observables (still local to a single moment in time) to be dependent on parameters.

**Comment by sabinala** [comment]
You're correct, I was thinking it would solve this problem but it just simplifies it. I'll add this.

**Comment by sabinala** [comment]
Closing as this converter is no longer necessary
